// Copyright 2023 Intrinsic Innovation LLC
// Intrinsic Proprietary and Confidential
// Provided subject to written agreement between the parties.

syntax = "proto3";

package intrinsic_proto.perception;

message Vector4f {
  float x = 1;
  float y = 2;
  float z = 3;
  float w = 4;
}

message Vector3f {
  float x = 1;
  float y = 2;
  float z = 3;
}

message Vector2f {
  float x = 1;
  float y = 2;
}
